docs/src/demo/Sync.tsx

```diff
@@ -13,7 +17,171 @@ export default () => {
     flicking.camera.lookAt(flicking.camera.range.min + flicking.camera.rangeDiff * progress);
   };
 
-  return <div className="has-background-grey-lighter p-4">
+  const reactSourceCode = `export default (): React.FC => {
+  const flicking0 = useRef<Flicking>();
```

Comment:
In the past, there was a question that a javascript user copied and pasted the typescript code and it didn't work.

Since angular is forced typescript, there is no problem, but in react, how about omitting the type for javascript users?

Reply:
Oh, I didn't think that.
I'll fix that to use js instead then